Homework_7.py

```diff
@@ -0,0 +1,95 @@
+# Homework 7
+
+import csv  # import module for work with csv
+import os  # import module for ability to delete files
+from Homework_6 import *  # import previous homeworks (5-6)
+
+
+class CountWords:
+    def parse_file(self, list_with_words):
```

Comment:
It is a static method. It means, that it is not referring to any property of the class object or any other method of the class.
Such methods have decorator `@staticmethod` and do not take self as an argument.

More: https://www.programiz.com/python-programming/methods/built-in/staticmethod